#163254259 implement feedback gotten on User Interface

tunedev opened the pull request:
#### What does this PR do?
Improves User interface of the app
#### Description of Task to be completed?
Improves form feel and makes markup more attractive
#### How should this be manually tested?
click on the [link](https://swisskid95.github.io/Questioner/UI/), and view the new ui markup.
#### What are the relevant pivotal tracker stories?
[#163254259](https://www.pivotaltracker.com/story/show/163254259)

tunedev committed:
[chore #163149240] implement feedback gotten on user interface

tunedev committed:
chore review: implement feedback gotten on user interface

chukwuemekachm reviewed (changes requested):
Please modify your commit message to the specified convention.

chukwuemekachm reviewed (changes requested):
You are yet to update your commit message

tunedev committed:
[chore #163254259]implement feedback on User interface of this app
Merge branch 'ch-Implement-feedback-163254259' of https://github.com/swisskid95/Questioner into ch-Implement-feedback-163254259
